lib/binding_rust/lib.rs

```diff
@@ -1,22 +1,32 @@
 #![doc = include_str!("./README.md")]
-
+#![cfg_attr(not(feature = "std"), no_std)]
 pub mod ffi;
 mod util;
 
+#[cfg(not(feature = "std"))]
+extern crate alloc;
```

Comment:
Is this required?

Reply:
yes, alloc crate not included by default, we need include it by hand.

ref: https://docs.rust-embedded.org/book/collections/index.html#using-alloc

> [!NOTE]
> rust seems has no `#![feature(alloc)]` at this time.